pkg/k8sutil/resource.go

```diff
@@ -28,7 +28,7 @@ func Reconcile(log logr.Logger, client runtimeClient.Client, desired runtimeClie
 	default:
 		var key runtimeClient.ObjectKey
 		key = runtimeClient.ObjectKeyFromObject(current)
-		log = log.WithValues("kind", desiredType, "name", key.Name)
+		log.Debug("reconciling", zap.Any("kind", desiredType), zap.Any("name", key.Name))
```

Comment:
Should these `zap.Any()` method calls be `zap.String()`? It may not make a difference. Just checking 😄

Reply:
We may as well make them consistent - especially as they're output to JSON or console.
I'll convert them now